datahub-web-react/src/app/homeV2/reference/header/GreetingText.tsx

```diff
@@ -34,6 +40,22 @@ export const GreetingText = ({ role }: { role?: string | null }) => {
     const userContext = useUserContext();
     const greetingText = getGreetingText();
     const { user } = userContext;
+
+    const showNavBarRedesign = useShowNavBarRedesign();
+
+    if (showNavBarRedesign) {
```

Comment:
So now we are just completley removing the user image, right?

Reply:
correct user image is gone-zo